packages/storybook-utils/src/source-code-generator.spec.ts

```diff
@@ -234,3 +236,23 @@ test.each([
   const docgenInfo = parseDocgenInfo({ __docgenInfo });
   expect(docgenInfo.eventNames).toStrictEqual(eventNames);
 });
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+test.each<{ fn: (...args: any[]) => unknown; expectedNames: string[] }>([
+  { fn: () => ({}), expectedNames: [] },
+  { fn: (a) => ({}), expectedNames: ["a"] },
```

Comment:
can't you underscore the variables so the eslint errors don't come up?
```suggestion
  { fn: (_a) => ({}), expectedNames: ["_a"] },
```